include/grpcpp/ext/otel_plugin.h

```diff
@@ -142,6 +149,10 @@ class OpenTelemetryPluginBuilder {
   /// Registers a global plugin that acts on all channels and servers running on
   /// the process.
   absl::Status BuildAndRegisterGlobal();
+  /// Builds an open telemetry plugin, returns the plugin object when succeeded
+  /// or an error status when failed.
+  GRPC_MUST_USE_RESULT absl::StatusOr<std::shared_ptr<OpenTelemetryPlugin>>
+  Build();
```

Comment:
I'm thinking - mark this as an experimental API first. After we confirm that our users are happy with this, we can stabilize it.

Reply:
We should also add a comment saying that you can only Build once (irrespective  of whether `Build()` is used or `BuildAndRegisterGlobal()` is used

Reply:
Can we mark this as experimental as well please?

Reply:
done